bk-config/bin/create_beekeeper_cert.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
IMO you should have `-e` at the top of every script.  `#!/bin/bash -e`

I also don't see a problem with the `-x` also being enabled all the time `#!/bin/bash -ex`

Reply:
For `-e` I'd agree with you. `-x` is annoying when you have if-statements etc, as it floods stdout. I'll wrap `-x`/`+x` around the commands.